qcodes/tests/dataset/helper_functions.py

```diff
@@ -0,0 +1,33 @@
+from typing import Sequence, Tuple, Dict
+
+import numpy as np
+
+
+def verify_data_dict(data: Dict[str, Dict[str, np.ndarray]],
+                     parameter_names: Sequence[str],
+                     expected_names: Dict[str, Sequence[str]],
+                     expected_shapes: Dict[str, Sequence[Tuple[int, ...]]]) -> None:
+    """
+    Simple helper function to verify a dict of data
+
+    Args:
+        data: The data to verify
+        parameter_names: names of the parameters requested
```

Comment:
this is a bit confusing. "requested" for "checking", right? could you rephrase it please?